src/style.css

```diff
@@ -0,0 +1,70 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  background-color: #eee;
+}
+
+.header {
+  background-color: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  
+  height: 60px;
+  margin-top: 20px;
+  margin-left: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  position: relative;
+}
+
+.link__logo {
+  position: absolute;
+  left: 50px;
+}
+
+.nav {
+  height: 100%;
+}
+
+.nav__list {
+  height: 100%;
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+  margin: 0 50px 0 0;
+}
+
+.nav__item {
+  margin: 0 10px;
```

Comment:
better to use a left or right margin of 20px
![Screenshot 2021-04-01 at 10 26 41](https://user-images.githubusercontent.com/35385361/113258666-e6edd380-92d4-11eb-8817-59ff350c6478.png)
ul shouldn't contain any spaces on right and left sides

Reply:
paddings and margins on this img also redundant